pypeit/calibrations.py

```diff
@@ -652,7 +652,7 @@ def get_slits(self):
                                                                  bias=self.msbias, bpm=self.msbpm,
                                                                  dark=self.msdark)
                 self.edges = edgetrace.EdgeTraceSet(self.traceImage, self.spectrograph,
-                                                    self.par['slitedges'], bpm=self.msbpm,
+                                                    self.par['slitedges'], #bpm=self.msbpm,
```

Comment:
I seem to remember the bpm being important for constructing the Sobel image. Why is this now disbabled? Or is it not just not used for intstantiation?

Reply:
The bpm is now always pulled from the traceImage object.